modules/nodes-nodes-configuring-graceful-shutdown.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-activating-graceful-shutdown_{context}"]
+= Configuring graceful node shutdown
+
+To configure graceful node shutdown for a node, create a `KubeletConfig` custom resource (CR) to specify a grace period and shutdown profile for the nodes corresponding to a specific `MachineConfigPool`. If graceful node shutdown is not configured, the default grace period is 0 and the pod is forcefully terminated before `preStop` has run to completion.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Define a `KubeletConfig` CR with graceful node shutdown.
+. Create the `KubeletConfig` object:
++
+[source,terminal]
+----
+$ oc create -f <filename>
+----
+
+. Example KubeletConfig
```

Comment:
Could this step just be meaning does "Example KubletConfig" need to be here? It seems out of place.
```suggestion
. Apply the following settings to configure graceful node shutdown.
```